internal/buffer/save.go

```diff
@@ -24,6 +24,35 @@ import (
 // because hashing is too slow
 const LargeFileThreshold = 50000
 
+func resolvePath(filename string) (string, error) {
```

Comment:
There is one reason, why I left that behind:
With the rename we will use the original creation time stamp of the target file and will take over the one from the file which we renamed. The user could need these timestamps for further file filtering, which I don't like to remove.

Reply:
> With the rename we will use the original creation time stamp of the target file and will take over the one from the file which we renamed. The user could need these timestamps for further file filtering, which I don't like to remove.

I don't understand this at all. What does it mean "to take over the timestamp"? What "filtering"?

Reply:
Let me explain it with this snippet:
```shell
$ ls -lth --time=birth
-rw-rw-r-- 1 joeran joeran    0 17. Mai 07:13 new.txt
-rw-rw-r-- 1 joeran joeran    0 17. Mai 07:12 old.txt
$ ls -lth --time=mtime
-rw-rw-r-- 1 joeran joeran    0 17. Mai 07:13 new.txt
-rw-rw-r-- 1 joeran joeran    0 17. Mai 07:12 old.txt
$ echo "test" > new.txt 
$ ls -lth --time=birth
-rw-rw-r-- 1 joeran joeran    5 17. Mai 07:13 new.txt
-rw-rw-r-- 1 joeran joeran    0 17. Mai 07:12 old.txt
$ ls -lth --time=mtime
-rw-rw-r-- 1 joeran joeran    5 17. Mai 07:14 new.txt
-rw-rw-r-- 1 joeran joeran    0 17. Mai 07:12 old.txt
$ mv new.txt old.txt 
$ ls -lth --time=birth
-rw-rw-r-- 1 joeran joeran    5 17. Mai 07:13 old.txt
$ ls -lth --time=mtime
-rw-rw-r-- 1 joeran joeran    5 17. Mai 07:14 old.txt
```

By "filtering" I was referring to the use case a user likes to search/filter for files created instead of modified at a certain time. We will overwrite the files birth date in the moment we rename the file, which we shouldn't do.

Reply:
Thanks, I wasn't even aware that file creation timestamps are now a thing in Linux.

So, if we implement rename, we need not just to copy the file's owner, mode and xattrs (as I [mentioned](https://github.com/zyedidia/micro/pull/3273#discussion_r1586295200) earlier) but also copy at least it creation time...

Aaaa... got it. These creation timestamps are immutable in Linux, we cannot just set them.

Ok, I got your point: you don't want to implement rename at all, for this reason. Yeah, it makes sense (and there may be even more reasons, which we don't realize yet).

So, we can just remove symlink resolving now, right?

Reply:
> So, we can just remove symlink resolving now, right?

Yep, due to the fact we don't need the full absolute path any longer (no renaming), it's superfluous.
We can simply work with the given paths (and symlinks too).